scripts/lib/mkcloud-libvirt.sh

```diff
@@ -306,11 +306,9 @@ function libvirt_do_onhost_deploy_image()
     local disk=$3
 
     [[ $clouddata ]] || complain 108 "clouddata IP not set - is DNS broken?"
-    pushd /tmp
-    local wgetcachemode=-N
-    [[ $want_cached_images = 1 ]] && wgetcachemode=-nc
```

Comment:
I don't quite understand the use case you mention: The `-nc` flag will cause wget to keep copies of previously downloaded images. How and why would I use those as a remote worker? What am I missing here?